tests/test_international_days.py

```diff
@@ -0,0 +1,31 @@
+from datetime import date
+
+import pytest
+
+from app.database.models import InternationalDays
+from app.internal.international_days import get_international_days
+from tests.utils import create_model, delete_instance
+
+DATE = date(2021, 6, 1)
+DAY = "Hamburger day"
```

Comment:
Is there a calendar day with no international day? What about day in which multiple international days occurs?

Reply:
I created the json so there should not be a day without an international day, but the test  `test_international_day_per_day_no_international_days` supposed to check it anyway.
For the multiple international days problem i always taking the first item in the list that came from the DB and in the json i created if I have multiple international days the are in the one string in the dictionary separated by comma.

Reply:
So you can create a test to check that every day in the year actually has an international day? :)
Could you also create a check for multiple "international days" in the same date?

Reply:
fixed